### PR TITLE
native: reduce space within/between chat messages, fix keyboard offset in thread view

### PR DESCRIPTION
**Pull request by patosullivan**
Fixes TLON-2055
Fixes TLON-2073

### CODE REVIEW

File: packages/ui/src/components/MessageInput/index.tsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { useState } from 'react';
 import { TextArea } from '../../core';
 import { MessageInputContainer, MessageInputProps } from './MessageInputBase';
 
+export const DEFAULT_MESSAGE_INPUT_HEIGHT = 44;
```
Comment (dnbrwstr) [inline packages/ui/src/components/MessageInput/index.tsx:6]: Can we actually apply this to the input to enforce it matching?

File: packages/ui/src/components/PostScreenView.tsx [hunk 2]
```diff
@@ -96,7 +96,9 @@ export function PostScreenView({
               <KeyboardAvoidingView
                 //TODO: Standardize this component, account for tab bar in a better way
                 behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
-                // keyboardVerticalOffset={70}
+                keyboardVerticalOffset={
+                  Platform.OS === 'ios' ? -bottom : DEFAULT_MESSAGE_INPUT_HEIGHT
```
Comment (dnbrwstr) [inline packages/ui/src/components/PostScreenView.tsx:100]: Does it make sense to extract this wrapper component and use across multiple screens? I'm guessing we have similar in other places?

**Review by dnbrwstr — APPROVED** [review]

File: packages/ui/src/components/PostScreenView.tsx (hunk 2 above)
Comment (patosullivan) [reply packages/ui/src/components/PostScreenView.tsx:100]: Looked for that, afaict this is the only place with this exact offset.

File: packages/ui/src/components/MessageInput/index.tsx (hunk 1 above)
Comment (patosullivan) [reply packages/ui/src/components/MessageInput/index.tsx:6]: We do, in /MessageInput/index.native.tsx

Comment (dnbrwstr) [reply packages/ui/src/components/MessageInput/index.tsx:6]: Ahh I see -- can index.native import the constant from this file then? Nbd if not though.

**Review by jamesacklin — APPROVED** [review]
Much denser. We'll refine more later but looks good to me!

File: packages/ui/src/components/MessageInput/index.tsx (hunk 1 above)
Comment (patosullivan) [reply packages/ui/src/components/MessageInput/index.tsx:8]: Now using the single definition.

Comment (patosullivan) [reply packages/ui/src/components/MessageInput/index.tsx:6]: Causes a require cycle, reverting for now.

Comment (dnbrwstr) [reply packages/ui/src/components/MessageInput/index.tsx:6]: Dang, thanks for giving it a try.

Comment (patosullivan) [reply packages/ui/src/components/MessageInput/index.tsx:6]: I'm fixing the keyboardavoidingview issue and it'll obviate the need for this anyway

**Pull request merged**